src/store/noteStore.ts

```diff
@@ -54,21 +56,20 @@ const createNote = createAsyncThunk(
   ) => {
     thunkAPI.dispatch(noteStore.actions.setIsLoading(true));
 
-    const newNoteId = filename as unknown as NoteId;
     const newNote: Note = {
-      id: newNoteId,
+      id: filename as unknown as NoteId,
```

Comment:
`as unknown` might not be needed

Reply:
It is because filename is of type `NoteName` :(